contracts/dtoken/tests/sim/scenarious.rs

```diff
@@ -286,7 +306,106 @@ fn scenario_02(){
 
     assert_failure(result, "Withdrawal operation is not allowed");
     
-    let user_balance: u128 = view_balance(&controller, d_user.account_id(), dtoken.account_id());
+    let user_balance: u128 = view_balance(&controller, Supply, d_user.account_id(), dtoken.account_id());
     assert_eq!(user_balance, 0, "Balance should be 0");
 
-}
\ No newline at end of file
+}
+
+#[test]
+fn scenario_03(){

```

Comment:
@dariacherniak 

1) please come up with some long but meaningful scenario name, something like 
```
fn scenario_repay_whole_amount
```
2) once you have checked some functionality you don't need to check it again in other scenarios, so if you have scenario where `utoken.mint` works correctly and if your scenario do not affect `utoken` you don't need to check of result of this function

3) I suppose you have a lot of boilerplate code with initializing state of contracts, in tests such pre-configuration called **Fixture**,

this scenario actually tests only dtoken::repay so all other actions are Fixutre to setup this scenario;
it seems okay to create something like 
scenario_repay_whole_amount_fixutre where to move deploying of accounts and initing balance